### ISSUE TITLE
Create a `NewIntegrationModal` component

### ISSUE THREAD

**Issue by ivicac** [bytechefhq/bytechef#613]
[Balsamiq mockup](https://balsamiq.cloud/spslzug/ph7qmda/rF477)

Leverage the [radix-ui Dialog component](https://www.radix-ui.com/docs/primitives/components/dialog#dialog).

It contains a potentially complex component (the tag selector), so if needed, create a separate task for it.

Once this is done, create an issue that handles the submitting of data to create the new integration.
